addons/website_event_track/views/event_track_templates_list.xml

```diff
@@ -117,13 +117,18 @@
     <!-- No tracks -->
     <t t-if="not tracks">
         <div class="col-12">
-            <div class="h2 mb-3">No track found.</div>
-            <div t-if="search_key" class="alert alert-info text-center">
-                <p class="m-0">We did not find any track matching your <strong t-out="search_key"/> search.</p>
+            <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                <div class="m-3">
+                    <t t-call="website_event.event_svg"/>
+                </div>
+                <h2 class="mb-2">No track found.</h2>
+                <p t-if="search_key">We could not find any track matching your <strong t-out="search_key"/> search.</p>
+                <p t-else="">We could not find any track at this moment.</p>
             </div>
-            <div t-else="" class="alert alert-info text-center" groups="event.group_event_user">
-                <a target="_blank" t-att-href="'/web?#action=website_event_track.action_event_track_from_event&amp;active_id=%s' % event.id">
-                    <p class="m-0">Schedule some tracks to get started</p>
+            <div class="o_not_editable d-flex flex-column align-items-center justify-content-between my-3" groups="event.group_event_user">
+                <a class="o_wevent_cta btn" target="_blank" t-att-href="'/web?#action=website_event_track.action_event_track_from_event&amp;active_id=%s' % event.id">
+                    <span class="fa fa-plus me-1"/>
```

Comment:
Convention: wherever it makes sense, code **inline content** like it will be rendered, so:
```xml
<a>
    <span class="icon"> My awesome text
</a>
```